AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -22,4 +22,12 @@
 
 - (instancetype)init;
 
+- (void)setChild:(id<ASLayoutable>)child;
```

Comment:
Perhaps worth some comments on children:
- All valid layout specs should have at least one child (is this true?)
- Base class implements handling of multiple children, and named children, as it is commonly required by subclasses (give a couple examples)
- However, not all layout specs may handle multiple children.  (Examples: inset node has one, background node has exactly two, which are specially identified)

Docs like this may not feel that critical, and it's hard to know the exact impact that they will have...but both for our team and others, I'm pretty sure they will help folks confirm their guesses and build confidence in their understanding as they explore the headers during normal implementation work.